src/app/features/swap-form/reducer.ts

```diff
@@ -106,19 +173,95 @@ export const slice = createSlice({
   },
   extraReducers: (builder) => {
     builder
-      .addCase(initialiseCashflowCalculator.pending, (state, {}) => {
-        state.cashflowCalculator.status = 'pending';
+      .addCase(initialiseCashflowCalculatorThunk.pending, (state) => {
+        state.cashflowCalculator.variableFactorStartNow = {
+          value: 0,
+          status: 'pending',
+        };
+      })
+      .addCase(initialiseCashflowCalculatorThunk.rejected, (state) => {
+        state.cashflowCalculator.variableFactorStartNow = {
+          value: 0,
+          status: 'error',
+        };
+      })
+      .addCase(initialiseCashflowCalculatorThunk.fulfilled, (state, { payload }) => {
+        state.cashflowCalculator.variableFactorStartNow = {
+          value: payload as number,
+          status: 'success',
+        };
+      })
+      .addCase(getFixedRateThunk.pending, (state) => {
+        state.fixedRate = {
+          value: 0,
+          status: 'pending',
+        };
+      })
+      .addCase(getFixedRateThunk.rejected, (state) => {
+        state.fixedRate = {
+          value: 0,
+          status: 'error',
+        };
+      })
+      .addCase(getFixedRateThunk.fulfilled, (state, { payload }) => {
+        state.fixedRate = {
+          value: payload as number,
+          status: 'success',
+        };
+      })
+      .addCase(getVariableRateThunk.pending, (state) => {
+        state.variableRate = {
+          value: 0,
+          status: 'pending',
+        };
+      })
+      .addCase(getVariableRateThunk.rejected, (state) => {
+        state.variableRate = {
+          value: 0,
+          status: 'error',
+        };
+      })
+      .addCase(getVariableRateThunk.fulfilled, (state, { payload }) => {
+        state.variableRate = {
+          value: (payload as number) * 100,
```

Comment:
why not multiple by 100 in the thunk return? I think that kind of multiplication should happen there. Here you just set it to store.

Reply:
[957310c](https://github.com/Voltz-Protocol/voltz-ui/pull/571/commits/957310cc8a0b03d2680b71d8f8ec5e98803f7150)